.github/actions/unit-tests/action.yml

```diff
@@ -16,7 +16,7 @@ runs:
     - name: run tests
       shell: bash
       run: |
-        python -Wd -m pytest -p no:randomly --ds=${{ env.settings_path }} ${{ env.unit_test_paths }}
+        python -Wd -m pytest -p no:randomly --ds=${{ env.settings_path }} ${{ env.unit_test_paths }} --cov=$(echo ${{ env.unit_test_paths }} | tr "," " ")
```

Comment:
The `tr` command is here to handle a comma-separated list. But if there is more than one, they will become space-separated, and then the option won't parse properly, will it?  The result will be something like `--cov=a b c`, which won't provide three values to the `--cov` option.

Maybe I've misunderstood something.

Reply:
Your command is `tr "," " "`.  That translates commas to spaces.  I think you want `tr " " ","`.  Do you have a way to test this action that will confirm it's working the way you need it to?

Reply:
Actually, you were right. the way I was doing converting space to comma but still space separated paths were working. So to confirm this, I converted this (old command which was causing issue before)`--cov=${{ env.unit_test_paths }}` into `--cov="${{ env.unit_test_paths }}"` (new command, by adding double quotes around), so this command is also working.

**Conclusion:** _`--cov` wants space separated paths string but that should be in double quotes_